backend/src/database/getSubGoals.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiError, OntologyEntity } from '../types/types';
+import DB from './index';
+import getSubGoals from './queries/getSubGoals';
+
+export default async (nodeId: string): Promise<OntologyEntity> => {
```

Comment:
This function doesn't really return an OntologyEntity. If you change "instancesOf" to "Subject" and "label" to "SubjectLabel" in the query, you can make a mapRecordToSubject similar to the existing mapRecordToObject to map the results to Array<OntologyEntity>. If you also want to include the description you should make a derived class. Just make sure the return type of the function matches the actual data.